### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by JosiasVasquez** [uroesch/CmderPortable#30]
Hello, I downloaded and extract the package, but when I execute that, the interface is not accessible with screen reader.
The screen reader only read me the window title, and then the panel word.
no more.
I don't know if you are programming the interface, or only do the portable package.
If you are programming the interface, I can help you to get accessibility, if not, could you give me the link to send comments to the developers.
Thanks a lot.

**Comment by uroesch** [comment]
Hello Josias,

Sad to hear that this also didn't work out for you. 

I only package the portable version and have generally no influence over the software itself. Any issue with the software would have to be addressed upstream. 

* For Cmder you can file issues here https://github.com/cmderdev/cmder/issues
* For the Issue with the [uroesch/GitPortable#114](windows terminal) you may do so here https://github.com/git-for-windows/git/issues

I will give the uroesch/GitPortable#114 another look and try to figure out what exactly the problem is there. But I don't know how fast I will come to a conclusion there.

cheers

**Comment by uroesch** [comment]
Accesibiltiy issue should be taken care of by resolving https://github.com/uroesch/GitPortable/issues/114

**Comment by JosiasVasquez** [comment]
Thanks, I will create an issue for this problem.
